astropy/io/misc/yaml.py

```diff
@@ -61,24 +59,16 @@
 import base64
 
 import numpy as np
+import yaml
 
 from astropy.time import Time, TimeDelta
 from astropy import units as u
 from astropy import coordinates as coords
 from astropy.utils import minversion
 from astropy.table import SerializedColumn
 
-try:
-    import yaml
-except ImportError:
-    raise ImportError('The PyYAML package is required for astropy.io.misc.yaml and must be 3.13 or later')
-else:
-    if not minversion(yaml, '3.13'):
```

Comment:
lint: since this change `minversion` is no longer used in this module, so its import can be removed.

Reply:
Done!